schemas/2019-04-01/autogeneratedResources.json

```diff
@@ -231,6 +231,15 @@
         },
         {
           "$ref": "https://schema.management.azure.com/schemas/2019-04-01/Microsoft.DevSpaces.json#/resourceDefinitions/controllers"
+        },
```

Comment:
`schemas/2019-04-01/deploymentTemplate.json` pulls in this file, so by including in both places this will cause a duplicate definition. Since some manual work was required to add "eventSubscriptions", please remove the changes to the "autogeneratedResources.json" file, and we can track this schema as manually created for now.